gl_autotool/pages/basic_page.py

```diff
@@ -1,71 +1,47 @@
-from playwright.sync_api import Page, Locator, ConsoleMessage
-from playwright.sync_api import sync_playwright
+from playwright.sync_api import Page, ConsoleMessage
 
 
 class Base:
     def __init__(self, page: Page):
-        """
-        Initialize basic page object
-        :param page: Playwright Page
-        """
         self.page = page
         self.context = page.context
 
         self.page.on('console', self._console_handler)
 
-    def _console_handler(self, msg: ConsoleMessage):
+    def _console_handler(self, msg: ConsoleMessage) -> None:
         """
         Handle console message
-        :param msg: console message object
+
+        Parameters
+        ----------
+        msg
```

Comment:
The original format was ok. In tis format, you also have to provide description of the param